include/bpf_helper_defs.h

```diff
@@ -473,3 +473,25 @@ EBPF_HELPER(size_t, bpf_strnlen_s, (const char* str, size_t str_size));
 #define memcpy_s bpf_memcpy
 #define memmove_s bpf_memmove
 #endif
+
+/**
+ * @brief Return time elapsed since boot in milliseconds including time while suspended.
+ * This function uses a lower resolution clock source than bpf_ktime_get_boot_ns, but is faster.
+ *
+ * @return Time elapsed since boot in milliseconds units.
```

Comment:
```suggestion
 * @return Time elapsed since boot in milliseconds.
```
grammar.  Yes `bpf_ktime_get_boot_ns()` doxygen was already equally bad and should be updated too.  In contrast, https://www.man7.org/linux/man-pages/man7/bpf-helpers.7.html has the right grammar "time elapsed since system boot, in nanoseconds."  Line 478 is fine.